Sample App/Use Cases/WidgetsUseCase.swift

```diff
@@ -74,16 +77,12 @@ class WidgetsUseCase: UIViewController {
     }
     
     private func setupEngagementSDK() {
-        
-        let sdk = EngagementSDK.init(clientID: clientID,
-                                     accessTokenStorage: self)
-        let config = SessionConfiguration(programID: programID)
-        session = sdk.contentSession(config: config, delegate: self)
-        EngagementSDK.logLevel = .verbose
+        sdk = EngagementSDK(clientID: clientID)
```

Comment:
This is deprecated - should use the EngagementSDK(config:) instead.

Reply:
Good point, it's fixed